flake.nix

```diff
@@ -78,7 +88,17 @@
           // builtins.mapAttrs (_: crane-lib.cargoDoc) crates
           # fmt
           // builtins.mapAttrs (_: crane-lib.cargoFmt) crates;
-
+        overlays.default = import ./nix/overlays.nix { inherit pkgs packages; };
+        nixosModules.default = import ./nix/genealogos-module.nix;
+        nixosConfigurations.genealogos = nixpkgs.lib.nixosSystem
```

Comment:
Is it common practice to include a whole NixOS configuration here? Is it some kind of basic example, or is it useful to deploy a simple genealogos "server" or something like that?

Reply:
I've renamed the vm to `genealogos-test`, hoping that makes it more clear what the purpose is.

It's not uncommon, I believe.